fe/fe-core/src/main/java/org/apache/doris/catalog/external/ExternalDatabase.java

```diff
@@ -107,21 +107,25 @@ public boolean isInitialized() {
     }
 
     public final synchronized void makeSureInitialized() {
-        if (!initialized) {
-            if (!Env.getCurrentEnv().isMaster()) {
-                // Forward to master and wait the journal to replay.
-                int waitTimeOut = ConnectContext.get() == null ? 300 : ConnectContext.get().getExecTimeout();
-                MasterCatalogExecutor remoteExecutor = new MasterCatalogExecutor(waitTimeOut * 1000);
-                try {
-                    remoteExecutor.forward(extCatalog.getId(), id);
-                } catch (Exception e) {
-                    Util.logAndThrowRuntimeException(LOG,
-                            String.format("failed to forward init external db %s operation to master", name), e);
-                }
-                return;
+        if (initialized && extCatalog.isInitialized()) {
+            return;
+        }
+        if (!extCatalog.isInitialized()) {
```

Comment:
I think we can move `extCatalog.isInitialized()` into `extCatalog.makeSureInitialized()`. Because `extCatalog.makeSureInitialized();` is a reentryable method